charts/gitlab/templates/secret.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
 type: Opaque
 data:
   gitlab-password: {{ required "Fill in .Values.password with `openssl rand -hex 16`" .Values.password | b64enc | quote }}
-  postgres-password: {{ required "Fill in .Values.global.gitlab.postgresPassword with `openssl rand -hex 16`" .Values.global.gitlab.postgresPassword | b64enc | quote }}
+  #postgres-password: {{ required "Fill in .Values.global.gitlab.postgresPassword with `openssl rand -hex 16`" .Values.global.gitlab.postgresPassword | b64enc | quote }}
```

Comment:
Do you want to keep this despite it's commented out?

Reply:
I honestly don't remember where this is from, `charts/gitlab/` shouldn't be changed here.